tests/assets/eks-pod-identity/pod-default.yaml

```diff
@@ -25,61 +32,49 @@ spec:
       - -c
       - |
         AUTH_TOKEN=$(cat $AWS_CONTAINER_AUTHORIZATION_TOKEN_FILE)
-        MAX_ATTEMPTS=7
-        INITIAL_DELAY=0.2  # 200ms
 
         DIMENSION_VALUE=$CLUSTER_NAME
         METRIC_MAX_RETRIES=5
         METRIC_RETRY_DELAY=1
 
-        # make 7 attempts on credential fetching with exponential retries, and calculate the time taken
+        # make an attempt on credential fetching, and calculate the time taken
```

Comment:
we are removing 7 retries and relying on underlying cli retires to succeed, right ?

Reply:
Yes